cli/bin/grain.js

```diff
@@ -75,11 +75,18 @@ class ForwardOption extends program.Option {
   }
 }
 
-program.forwardOption = function (flags, description, parser, defaultValue) {
+// Adds .forwardOption to commands. Similar to Commander's native .option,
+// but will forward the flag to the underlying program.
+program.Command.prototype.forwardOption = function (
```

Comment:
Out of curiosity, why was this change needed?

Reply:
It's not needed to fix the issue, but it's needed to be able to `.forwardOption` on the subcommand. We just decided it was cleaner to be consistent around `.forwardOption` versus `.addOption(new ForwardOption())`!